src/tool.rs

```diff
@@ -428,10 +428,7 @@ impl Tool {
 
     /// Supports using `--` delimiter to separate arguments and path to source files.
     pub(crate) fn supports_path_delimiter(&self) -> bool {
-        matches!(
-            self.family,
-            ToolFamily::Clang { .. } | ToolFamily::Msvc { clang_cl: true }
-        ) && !self.cuda
+        matches!(self.family, ToolFamily::Msvc { clang_cl: true }) && !self.cuda
```

Comment:
Nit: Would be nice to have a comment here that Homebrew doesn't support the flag, and that's why we don't use it on Clang.

Reply:
Thanks, added